internal/util/projutil/project_util.go

```diff
@@ -226,6 +246,15 @@ func IsOperatorGo() bool {
 }
 
 func IsOperatorAnsible() bool {
+	// If the project has the new layout we will check the type in the config file
+	if kbutil.HasProjectFile() {
+		cfg, err := kbutil.ReadConfig()
+		if err != nil {
+			log.Print(err, "Error reading config")
+		}
+		return PluginKeyToOperatorType(cfg.Layout) == OperatorTypeAnsible
+	}
+	// todo(camilamacedo86): remove when the legacy layout is no longer supported
```

Comment:
Will the roles dir no longer be in the new layout?

Reply:
The roles dir will be in the new layout, however, see:

- Follow up the same standard for all types shows very beneficial and increase the manutence ability
- We can init an Ansible/Helm project without scaffold an API which means that the roles dir would NOT exist.